app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ApplicationRecord
   ### ASSOCIATIONS ###
 
   has_one :address, as: :addressable, dependent: :destroy
+  has_many :orders, dependent: :destroy
```

Comment:
restrict with error. We should delete orders and payments from the system. 

If required to restrict any user to not use the system any more then we should keep another column on user like active

Reply:
Done